public/main.css

```
@@ -47,11 +46,10 @@ footer {
 	white-space: pre-line;
 	display: flex;
 	justify-content: center;
-	align-items: center;
+	align-items: 'center';
```

This is invalid css. The original is correct:
```suggestion
	align-items: center;
```